StructureDefinition/England-Task.xml

```diff
@@ -0,0 +1,304 @@
+      <definition value="**EPS Only**\n\nAn identifier to the original `prescription-order` **MUST** be present. Only the short form identifier needs to be supplied.\n\n```json\n&quot;groupIdentifier&quot;: {\n        &quot;system&quot;: &quot;https://fhir.nhs.uk/Id/prescription-order-number&quot;,\n        &quot;value&quot;: &quot;88AF6C-C81007-00001C&quot;\n    },\n```" />
+      <mustSupport value="true" />
+    </element>
+    <element id="Task.status">
+      <path value="Task.status" />
+      <definition value="These workflow tasks have `status` which indicates the Task status. \n\nThe following diagram reflects the &quot;typical&quot; state machine for Task. Note that not all states will be supported by all workflows and that some workflows may support additional transitions, including transitions from terminal states (e.g. back to &quot;in-progress&quot; from &quot;failed&quot; or &quot;completed&quot;).\n\n&lt;img src=&quot;https://www.hl7.org/fhir/task-state-machine.svg&quot;&gt;&lt;/img&gt;\n\nFor example: \n\nA patient requesting a repeat prescription would ask for task with code `fulfil` used with `103742009 | Renewal of prescription` and a status of `requested`.  \nOnce reviewed (by a clinician), the task status would be changed to `accepted`. When the task is then picked up (`in-progress`) and the related *MedicationRequest* is sent, the status would change to `completed`." />
+      <comment value="For **EPS**\n\nOnly `in-progress` is currently supported for reasonCode = 373784005 (Dispensing medication) and focus is a `dispense-notification`. \n\n`rejected` / `cancelled` / `failed` are supported for reasonCode = 33633005 (Prescription of drug) and focus is a `prescription-order`." />
+      <mustSupport value="true" />
+    </element>
+    <element id="Task.statusReason">
+      <path value="Task.statusReason" />
+      <definition value="**For EPS only**\n\n| reasonCode | focus | status | CodeSystem |\n|--|--|--|--|\n| 373784005 - Dispensing medication | `dispense-notification` | `cancelled` | [https://fhir.nhs.uk/CodeSystem/EPS-task-dispense-withdraw-reason](https://simplifier.net/resolve?target=simplifier&amp;scope=uk.nhsdigital.r4&amp;canonical=https://fhir.nhs.uk/CodeSystem/EPS-task-dispense-withdraw-reason) |\n| 33633005 - Prescription of drug | `prescription-order` | `rejected` / `cancelled` / `failed` |  [https://fhir.nhs.uk/CodeSystem/EPS-task-dispense-return-status-reason](https://simplifier.net/resolve?target=simplifier&amp;scope=uk.nhsdigital.r4&amp;canonical=https://fhir.nhs.uk/CodeSystem/EPS-task-dispense-return-status-reason) |\n\n```json\n&quot;statusReason&quot;: [{\n        &quot;coding&quot;:  [\n            {\n                &quot;system&quot;: &quot;https://fhir.nhs.uk/CodeSystem/EPS-task-dispense-withdraw-reason&quot;,\n                &quot;code&quot;: &quot;DA&quot;,\n                &quot;display&quot;: &quot;Dosage Amendments&quot;\n            }\n        ]\n    }]\n```" />
```

Comment:
Can you change this URL? 
1. https://fhir.nhs.uk/CodeSystem/EPS-task-dispense-withdraw-reason
2. ttps://fhir.nhs.uk/CodeSystem/EPS-task-dispense-return-status-reason
Updated URL is available in [MedicationDispense](https://github.com/NHSDigital/NHSEngland-FHIR-ImplementationGuide/tree/IOPS-1559-MedicationDispense)